process/sync/metablock_test.go

```diff
@@ -1177,6 +1177,7 @@ func TestMetaBootstrap_RollBackIsEmptyCallRollBackOneBlockOkValsShouldWork(t *te
 				//bytes represent a tx block body (strings are returns from txBlockUnit.Get which is also a stub here)
 				//copy only defined fields
 				obj = prevTxBlockBody
+				_ = obj
```

Comment:
this is masking an error.
please do a copy of the prevTxBlockBody fields into obj

Reply:
done.